dashboard/test/controllers/data_docs_controller_test.rb

```diff
@@ -22,4 +27,24 @@ class DataDocsControllerTest < ActionController::TestCase
   test_user_gets_response_for :create, params: -> {@test_params}, user: :student, response: :forbidden
   test_user_gets_response_for :create, params: -> {@test_params}, user: :teacher, response: :forbidden
   test_user_gets_response_for :create, params: -> {@test_params}, user: :levelbuilder, response: :success
+
+  # TODO [meg] : all should be able to see `show` page, once launched
+  # for now, only levelbuilder can see it
```

Comment:
again, this strategy is perfectly valid, but seems like unnecessary work to take these extra steps to protect these pages

Reply:
Sounds good –– will move these